recipe/meta.yaml

```diff
@@ -11,16 +11,16 @@ source:
         - EHapi.c.patch
 
 build:
-    number: 2
+    number: 3
     skip: True  # [win]
 
 requirements:
     build:
-        - hdf5 1.8.15.1
-        - zlib
+        - hdf5 1.8*
+        - zlib 1.2*
     run:
-        - hdf5 1.8.15.1
-        - zlib
+        - hdf5 1.8*
```

Comment:
If I get the pinning correctly I don't think this is good. `hdfeos5` is a `pynio` dependency and `pynio` needs to install `libnecdf` from the default channel and hence `hdf5 1.8.15.1`